readme.md

```diff
@@ -374,10 +375,12 @@ async function main() {
     .use(remarkPresetLintConsistent)
     // Few recommended rules.
     .use(remarkPresetLintRecommended)
-    // `remark-lint-list-item-indent` is configured with `tab-size` in the
-    // recommended preset, but if we’d prefer something else, it can be
-    // reconfigured:
-    .use(remarkLintListItemIndent, 'space')
+    .use({
+      // `remark-lint-list-item-indent` is configured with `tab` in the
+      // recommended preset, but if we’d prefer something else, it can be
+      // reconfigured:
+      settings: {listItemIndent: 'one'}
+    })
```

Comment:
I believe the prose here does not explain what’s happening correctly. The rule is not configured in the preset. The preset defines a shared setting that the rule and remark-stringify respond to.

Reply:
- [x] I've edited the prose:
> `settings.listItemIndent` is set to `tab` in the recommended preset, but you can change it if you’d prefer something else: